Mathlib/Data/IsROrC/Basic.lean

```diff
@@ -712,6 +712,9 @@ theorem norm_natCast (n : ℕ) : ‖(n : K)‖ = n := by
 theorem norm_ofNat (n : ℕ) [n.AtLeastTwo] : ‖(no_index (OfNat.ofNat n) : K)‖ = OfNat.ofNat n :=
   norm_natCast n
 
+lemma norm_nsmul [NormedAddCommGroup E] [NormedSpace K E] (n : ℕ) (x : E) : ‖n • x‖ = n • ‖x‖ := by
```

Comment:
Please make `K` explicit in this formula. It doesn't appear on either side of the formula.